frontend/src/components/common/NavBar.tsx

```diff
@@ -0,0 +1,178 @@
+        justifyContent="space-between"
+        marginLeft="80px"
+        marginRight="80px"
+        marginTop="14px"
+        marginBottom="14px"
+      >
+        <Image
+          src={FONIcon}
+          alt="FON icon"
+          display="inline"
+          width="40px"
+          height="40px"
+        />
+        {authenticatedUser ? (
+          <>
+            <Tabs
+              variant="unstyled"
+              onChange={navigate}
+              defaultIndex={getIndex as any}
+              width="750px"
+              alignContent="center"
+            >
+              <TabList justifyContent="space-evenly">
+                <Tab
+                  id="Camps"
+                  _selected={{
+                    color: COLOR.primary.green[100],
+                    fontWeight: FontWeights.BOLD,
+                  }}
+                >
+                  <FontAwesomeIcon icon={faCampground} />
+                  &nbsp; Camps
+                </Tab>
+                {authenticatedUser.role === Role.ADMIN && (
+                  <>
+                    <Tab
+                      id="Global Forms"
+                      _selected={{
+                        color: COLOR.primary.green[100],
+                        fontWeight: FontWeights.BOLD,
+                      }}
+                    >
+                      <FontAwesomeIcon icon={faPenToSquare} />
+                      &nbsp; Global Forms
+                    </Tab>
+                    <Tab
+                      id="Access Control"
+                      _selected={{
+                        color: COLOR.primary.green[100],
+                        fontWeight: FontWeights.BOLD,
+                      }}
+                    >
+                      <LockIcon />
+                      &nbsp; Access Control
+                    </Tab>
+                  </>
+                )}
+              </TabList>
+            </Tabs>
+          </>
+        ) : null}
+        {authenticatedUser ? (
+          <>
+            <Popover placement="bottom-start" matchWidth>
+              {({ isOpen }) => (
+                <>
+                  <PopoverTrigger>
+                    <Button
+                      rightIcon={
+                        isOpen ? <ChevronUpIcon /> : <ChevronDownIcon />
+                      }
+                      bg="#FFF"
+                      _hover={{
+                        color: "none",
+                      }}
+                      _active={{
+                        color: "none",
+                      }}
+                      _focus={{
+                        color: "none",
+                      }}
+                    >
+                      {authenticatedUser?.firstName}{" "}
+                      {authenticatedUser?.lastName}
+                    </Button>
+                  </PopoverTrigger>
+                  <PopoverContent width="inherit">
+                    <PopoverBody as={Button} bg="#FFF" onClick={onLogOutClick}>
```

Comment:
For this, since it's a background and it's white I believe we can just use `bg="background.white.100"`